aws/resource_aws_workspaces_workspace_test.go

```diff
@@ -401,32 +401,44 @@ func testAccCheckAwsWorkspacesWorkspaceExists(n string, v *workspaces.Workspace)
 func testAccAwsWorkspacesWorkspaceConfig_Prerequisites(rName string) string {
 	return composeConfig(
 		testAccAwsWorkspacesDirectoryConfig_Prerequisites(rName),
-		`
+		fmt.Sprintf(`
 data "aws_workspaces_bundle" "test" {
   bundle_id = "wsb-bh8rsxt14" # Value with Windows 10 (English)
 }
 
 resource "aws_workspaces_directory" "test" {
   directory_id = aws_directory_service_directory.main.id
+
+  tags = {
+    Name = "tf-testacc-workspaces-directory-%[1]s"
+  }
 }
-`)
+`, rName))
 }
 
 func testAccWorkspacesWorkspaceConfig(rName string) string {
-	return testAccAwsWorkspacesWorkspaceConfig_Prerequisites(rName) + `
+	return composeConfig(
+		testAccAwsWorkspacesWorkspaceConfig_Prerequisites(rName),
+		fmt.Sprintf(`
 resource "aws_workspaces_workspace" "test" {
   bundle_id    = data.aws_workspaces_bundle.test.id
   directory_id = aws_workspaces_directory.test.id
 
   # NOTE: WorkSpaces API doesn't allow creating users in the directory.
   # However, "Administrator"" user is always present in a bare directory.
   user_name = "Administrator"
+
+  tags = {
+    Name = "tf-testacc-workspaces-workspace-%[1]s"
+  }
```

Comment:
The `TestAccAwsWorkspacesWorkspace_basic` acceptance test needs to be updated with this change as well:

```
=== CONT  TestAccAwsWorkspacesWorkspace_basic
TestAccAwsWorkspacesWorkspace_basic: resource_aws_workspaces_workspace_test.go:64: Step 1/2 error: Check failed: 1 error occurred:
  * Check 15/15 error: aws_workspaces_workspace.test: Attribute 'tags.%' expected "0", got "1"
--- FAIL: TestAccAwsWorkspacesWorkspace_basic (1667.19s)
```

Reply:
Sorry for that oversight, fixed 🙇